src/sflow/hsflowd/Makefile

```diff
@@ -0,0 +1,21 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = $(HSFLOWD)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	rm -fr ./host-sflow
+	git clone https://github.com/sflow/host-sflow
+
+	pushd ./host-sflow
+	git checkout -b sflow -f 08c00e2
```

Comment:
do we have a release version?

Reply:
Not yet. This will be updated once we have a release version.